externs/browser/html5.js

```diff
@@ -3570,9 +3570,64 @@ HTMLDataListElement.prototype.options;
  */
 function HTMLOutputElement() {}
 
-// TODO(jakubvrana): Add HTMLOutputElement properties.
+/**
+ * @const
+ * @type {!DOMTokenList}
+ */
+HTMLOutputElement.prototype.htmlFor;
+
+/**
+ * @type {HTMLFormElement}
+ */
+HTMLOutputElement.prototype.form;
+
+/**
+ * @type {string}
+ */
+HTMLOutputElement.prototype.name;
+
+/**
+ * @const
+ * @type {!string}
+ */
+HTMLOutputElement.prototype.type = "output";
```

Comment:
This was intentional and I saw similar stuff (a function with an actual function body that returns a string, for example). This is a constant string in a read only attribute. Why is this problematic?